### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by DominikPinsel**
Signed-off-by: Dominik-Pinsel <dominik.pinsel@daimler.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,20 +53,26 @@
 
 ## About The Project
 
-The project provides pre-built control- and data-plane [docker](https://www.docker.com/) images and [helm](https://helm.sh/) charts of the [Eclipse DataSpaceConnector Project](https://github.com/eclipse-edc/Connector).
+The project provides pre-built control- and data-plane [docker](https://www.docker.com/) images and [helm](https://helm.sh/) charts of the [Eclipse DataSpace Components Project](https://github.com/eclipse-edc/Connector).
```
Comment (ndr-brt) [inline README.md:56]: I'd use "Eclipse Data Space Components Connector", or, better "Eclipse EDC Connector"

File: README.md [hunk 2]
```diff
@@ -53,20 +53,26 @@
...
-The eclipse data space connector is split up into Control-Plane and Data-Plane, whereas the Control-Plane functions as administration layer
-and has responsibility of resource management, contract negotiation and administer data transfer. 
+The eclipse data space connector is split up into Control-Plane and Data-Plane, whereas the Control-Plane functions as administration layer and has responsibility of resource management, contract negotiation and administer data transfer. 
 The Data-Plane does the heavy lifting of transferring and receiving data streams.
 
 Depending on your environment there are different derivatives of the control-plane prepared:
 
 * [edc-controlplane-memory](edc-controlplane/edc-controlplane-memory) with dependency onto
     * [Azure Key Vault](https://azure.microsoft.com/en-us/services/key-vault/#product-overview)
+* [edc-controlplane-pomemory-hashicorp-vault](edc-controlplane/edc-controlplane-memory) with dependency onto
```
Comment (ndr-brt) [inline README.md:74]:
```suggestion
* [edc-controlplane-memory-hashicorp-vault](edc-controlplane/edc-controlplane-memory) with dependency onto
```

File: README.md [hunk 3]
```diff
@@ -53,20 +53,26 @@
...
+- In the **EDC** repository the actual application is developed. So _everything_ concerning the applications itself (questions/issues/discussions regarding the EDC domain or extensions) should be placed there. 
+- In the **TXDC** repository the Docker Images and Helm Charts are developed. So _everything_ concerning the image and deployment should be placed there.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 ## Inventory
 
-The eclipse data space connector is split up into Control-Plane and Data-Plane, whereas the Control-Plane functions as administration layer
-and has responsibility of resource management, contract negotiation and administer data transfer. 
+The eclipse data space connector is split up into Control-Plane and Data-Plane, whereas the Control-Plane functions as administration layer and has responsibility of resource management, contract negotiation and administer data transfer.
```
Comment (ndr-brt) [inline README.md:67]:
```suggestion
The Eclipse EDC Connector is split up into Control-Plane and Data-Plane, whereas the Control-Plane functions as administration layer and has responsibility of resource management, contract negotiation and administer data transfer. 
```

**Comment by florianrusch-zf** [comment]
I closed this PR, because we already moved to [eclipse-tractusx/tractusx-edc](https://github.com/eclipse-tractusx/tractusx-edc/)

**Pull request closed without merging**